app/config.py

```diff
@@ -61,7 +61,7 @@ class Settings(BaseSettings):
     "formatters": {"json": {"()": SafeJsonFormatter}},  # Custom formatter class
     "handlers": {
         "console": {
-            "level": "DEBUG",
+            "level": settings.LOG_LEVEL,
```

Comment:
```suggestion
            "level": "DEBUG",
```

`console` needs to log everything. log level is defined in the loggers, not in the handler